mealie/routes/admin/admin_invitations.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import APIRouter, Depends
+
+from mealie.routes._base import BaseAdminController, controller
+from mealie.schema.household.invite_token import (
+    ReadInviteToken,
+)
+from mealie.schema.make_dependable import make_dependable
+from mealie.schema.response.pagination import PaginationQuery
+
+router = APIRouter(prefix="/invitations")
+
+
+@controller(router)
+class AdminInvitationsController(BaseAdminController):
+    @router.get("", response_model=list[ReadInviteToken])
+    def get_all(
+        self,
+        q: PaginationQuery = Depends(make_dependable(PaginationQuery)),
+    ):
+        """Get all invite tokens"""
+        return self.repos.group_invite_tokens.page_all(pagination=q).items
```

Comment:
We should return the pagination object as-is instead of the items. It should be:

```python
response = self.repos.group_invite_tokens.page_all(pagination=q)

response.set_pagination_guides(router.url_path_for("get_all"), q.model_dump())
return response
```

This will also need to be updated on the frontend to accept the pagination response (rather than the list of invitations)